man/man1/p11sak.1.in

```diff
@@ -1859,7 +1859,12 @@ is used. If
 .B PKCS11_SO_PIN
 is not set, then you will be prompted for the SO PIN. SO can only access public
 token objects
-.RB ( CKA_PRIVATE = FALSE ).
+.RB ( CKA_PRIVATE = FALSE ),
+but SO can set attribute
+.B CKA_TRUSTED
+to
+.B TRUE
+for secret key, public key, and certificate objects.
```

Comment:
Hmm ... also plural for keys? Like "... for secret keys, public keys, and certificate objects' ?

Reply:
hm, well, the 'objects' was meant for all 3 of them....
Not sure how to make this clear. Usually we would use a dash like
`for secret key-, public key-, and certificate-objects.` but this does not look right either....

Probably so?
`for secret key objects, public key objects, and certificate objects.`

Reply:
ok, already changed and force pushed.